server/utils/suggestionHelpers.ts

```diff
@@ -2,77 +2,68 @@ import { PrisonerSearchForm } from 'express-session'
 
 export interface Suggestion {
   type: string
-  term: string
   val: string
-}
-
-export interface SuggestionFields {
-  firstName: Suggestion[]
-  lastName: Suggestion[]
-  age: Suggestion[]
+  query: string
```

Comment:
ended up with both `val` and `query` as for swaps need to modify two fields (first and last name) as well as providing text containing both fields